proto/meta.proto

```diff
@@ -259,6 +259,7 @@ message Relation {
     catalog.Index index = 4;
     catalog.View view = 5;
     catalog.Function function = 6;
+    catalog.Connection connection = 7;
```

Comment:
Is `Connection` a relation? Sounds weird to me. 🤔

Reply:
+1, and it seems that `Connection` are less likely to be updated with other relations, and it is better to notify function-related updates separately.

Reply:
fixed :)